http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -263,49 +284,26 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
       excludeStatuses = if (includeDeleted) List.empty else List(RuntimeStatus.Deleted)
       creatorOnly <- F.fromEither(processCreatorOnlyParameter(userInfo.userEmail, params, ctx.traceId))
 
-      // prevent misbehaved user input, labelMap keys and values can not have a single quote
-      // keys and values are inserted directly into a SQL string which could cause sql injection
-      // story #IA-xxx will refactor the SQL string into Slick
-      _ <- F.raiseWhen(
-        labelMap.values.exists(value => value.contains("'")) || labelMap.keys.exists(key => key.contains("'"))
-      )(
-        BadRequestException(s"Invalid query parameter (single quote not allowed)", Some(ctx.traceId))
-      )
-
+      authorizedIds <- getAuthorizedIds(userInfo, creatorOnly)
       runtimes <- RuntimeServiceDbQueries
-        .listRuntimes(labelMap, excludeStatuses, creatorOnly, cloudContext)
+        .listRuntimes(
+          // Authorization scopes
+          ownerGoogleProjectIds = authorizedIds.ownerGoogleProjectIds,
+          ownerWorkspaceIds = authorizedIds.ownerWorkspaceIds,
+          readerGoogleProjectIds = authorizedIds.readerGoogleProjectIds,
+          readerRuntimeIds = authorizedIds.readerRuntimeIds,
+          readerWorkspaceIds = authorizedIds.readerWorkspaceIds,
+          // Filters
+          excludeStatuses = excludeStatuses,
+          creatorEmail = creatorOnly,
+          cloudContext = cloudContext,
+          labelMap = labelMap
+        )
```

Comment:
v1 listRuntimes will now return *all visible runtimes* including Azure. Is this okay? If this is not desired I can easily restrict by emptying the `*workspaceIds` params.